package.json

```diff
@@ -13,13 +13,26 @@
   },
   "homepage": "https://github.com/alphagov/govuk-design-system#readme",
   "scripts": {
+    "build": "node scripts/build.js",
     "lint": "gulp scss:lint"
   },
   "dependencies": {
     "@govuk-frontend/all": "^0.0.18-alpha"
   },
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-sass-lint": "^1.3.4"
+    "gulp-sass-lint": "^1.3.4",
+    "jstransformer-markdown": "^1.2.1",
+    "jstransformer-nunjucks": "^0.5.0",
+    "metalsmith": "^2.3.0",
+    "metalsmith-in-place": "^3.0.1",
+    "metalsmith-layouts": "^1.8.1",
+    "metalsmith-markdown": "^0.2.1",
+    "metalsmith-metallic": "^2.0.2",
+    "metalsmith-permalinks": "^0.5.0",
+    "metalsmith-rollup": "^0.2.0",
+    "metalsmith-sass": "^1.4.0",
+    "nunjucks": "^3.0.1",
+    "sass-mq": "^3.3.2"
```

Comment:
sass-mq is already included by govuk-frontend so we probably don't need to include it again.

Reply:
removed